core/extrinsics_submission_service/extrinsic_submission_proxy.cpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "extrinsics_submission_service/extrinsic_submission_proxy.hpp"
+
+namespace kagome::service {
+  ExtrinsicSubmissionProxy::ExtrinsicSubmissionProxy(
+      sptr<ExtrinsicSubmissionApi> api)
+      : api_(std::move(api)) {}
+
+  std::vector<uint8_t> ExtrinsicSubmissionProxy::submit_extrinsic(
+      std::vector<uint8_t> bytes) {
```

Comment:
also here. maybe use a struct Extrinsic?

Reply:
we cannot use non-standard types in proxy, the meaning of it is to convert between types that can be bind by jsonrpc-lean library and our primitives.

Reply:
don't think it makes sense